test/uk/gov/hmrc/cdsimportsddsfrontend/controllers/DeclarationController_goodsIdentificationSpec.scala

```diff
@@ -92,7 +108,15 @@ with Scenarios with FutureAwaits with BeforeAndAfterEach {
         "goodsIdentification.description" -> Seq(""),
         "goodsIdentification.typeOfPackages" -> Seq(""),
         "goodsIdentification.numberOfPackages" -> Seq(""),
-        "goodsIdentification.shippingMarks" -> Seq("")
+        "goodsIdentification.shippingMarks" -> Seq(""),
```

Comment:
i think we found that all these `"xxxx" -> Seq("")` mappings are redundant - you can just pass the mandatory form data (currently `declarationTypeFormData`) and get the same result

Reply:
Ok. Will attempt to remove.